variables.tf

```diff
@@ -4,9 +4,10 @@ variable "profile" {
   default     = "default"
 }
 
-variable "account_id" {
-  type        = string
-  description = "The AWS account id to be used to create resources."
+variable "allowed_account_ids" {
+  type        = list(string)
+  description = "The list of account ids in which creation of resources is allowed."
+  default     = []
```

Comment:
Does empty array mean "all accounts" or "no accounts"?

Reply:
I asked myself this question as well and tried it out explicitly: It means "all accounts", if left unspecified resources get created in the account that is specified in the used profile.

Reply:
When we really remove the provider configuration from this repo (see https://github.com/dspace-group/simphera-reference-architecture-aws/pull/108#discussion_r1338839947) we would not need allowed_account_ids anymore.

Reply:
Yes, i have removed it again